engine/src/execution_graph/logic_controllers/BatchJoinProcessing.h

```diff
@@ -469,6 +470,13 @@ class JoinPartitionKernel :public kernel {
 		if (left_batch == nullptr || left_batch->num_columns() == 0){
 			std::cout<<"ERROR JoinPartitionKernel has empty left side and cannot determine join column indices"<<std::endl;
 		}
+
+		std::pair<bool, uint64_t> total_in = this->query_graph->get_estimated_input_rows_to_kernel(this->kernel_id);
+		std::cout<<"JoinPartitionKernel get_estimated_input_rows_to_kernel: "<<total_in.second<<std::endl;
```

Comment:
remove couts

Reply:
I absolutely will. This is still a WIP that is why its there